app/models/provider_key.py

```diff
@@ -19,6 +19,7 @@ class ProviderKey(BaseModel):
         String, nullable=True
     )  # Allow custom base URLs for some providers
     model_mapping = Column(JSON, nullable=True)  # JSON dict for model name mappings
+    billable = Column(Boolean, nullable=False, default=False)
```

Comment:
Shall we add a "billable" annotation at the model level instead of the provider level?